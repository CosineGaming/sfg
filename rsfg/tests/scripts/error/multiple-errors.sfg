fn main()
	first_error

fn two()
	second_error

fn three()
	if third_error
		fourth_error

