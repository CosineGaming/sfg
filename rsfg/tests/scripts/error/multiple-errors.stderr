[ERROR] expected (, =, assignment, got newline at 2:13
[ERROR] expected (, =, assignment, got newline at 5:14
[ERROR] expected (, =, assignment, got newline at 9:15